ouroboros-network-framework/test/Test/Ouroboros/Network/Server2.hs

```diff
@@ -716,7 +717,7 @@ unidirectionalExperiment
 unidirectionalExperiment timeouts snocket socket clientAndServerData = do
     nextReqs <- oneshotNextRequests clientAndServerData
     withInitiatorOnlyConnectionManager
-      "client" timeouts snocket Nothing nextReqs
+      "client" timeouts nullTracer snocket Nothing nextReqs
       $ \connectionManager ->
         withBidirectionalConnectionManager "server" timeouts nullTracer
                                            snocket socket Nothing
```

Comment:
Maybe this patch as well?

Reply:
Are you pointing to 5aa19bcc75b1e1d231b329c31296a1313d7ea9b6? Then no, I think it's nice to have it as a separate commit. It could be a separate test case.